src/scripts/components/NotificationHubIcon.js

```diff
@@ -1,15 +1,36 @@
 import { __ } from '@wordpress/i18n';
 import { useShortcut } from '@wordpress/keyboard-shortcuts';
+import classNames from 'classnames';
 
-export const NotificationHubIcon = ( { toggle, isActive } ) => {
+/**
+ * Notification icon UI component
+ *
+ * @param {Object}    Props
+ * @param {Function}  Props.toggle   - Toggle the drawer on and off
+ * @param {boolean}   Props.isActive - the drawer active state
+ * @param {string[]=} Props.classes  - the icon class name (defaults to [ 'dashicons-bell' ])
+ * @return {JSX.Element} - the Notification icon
```

Comment:
JSDocs shouldn't have the `dash` it is added by the TypeScript language server so it gets doubled up if you have then in the comment.
```suggestion
 * @param {Function}  Props.toggle   Toggle the drawer on and off.
 * @param {boolean}   Props.isActive Predicate of whether or not the drawer is in an active state.
 * @param {string[]=} Props.classes  The icon class names (defaults to [ 'dashicons-bell' ])
 * @return {JSX.Element} - the Notification icon
```

Reply:
"languageTools" a plugin on my ide suggests this change // marks the fix as an error... @johnhooks, since you are a native speaker what do you say? Do I apply the suggestion?

![image](https://user-images.githubusercontent.com/8550908/232237697-657e2296-0049-47dc-8625-2ffb7d353c57.png)

Reply:
Yep, apply that suggestion.